flows/events/base_test.go

```diff
@@ -487,6 +490,6 @@ func TestWebhookCalledEventBadUTF8(t *testing.T) {
 	event := events.NewWebhookCalled(call, flows.CallStatusSuccess, "")
 
 	assert.Equal(t, "http://temba.io/", event.URL)
-	assert.Equal(t, "HTTP/1.0 200 OK\r\nContent-Length: 2\r\n\r\n\xa0\xa1", event.Response)
-	assert.False(t, utf8.Valid([]byte(event.Response)))
+	assert.Equal(t, "HTTP/1.0 200 OK\r\nContent-Length: 2\r\n\r\n...", event.Response)
+	assert.True(t, utf8.ValidString(event.Response))
```

Comment:
anything written into an event should be serializable as JSON, and should be valid utf-8.. so if we get a big binary payload, we replace it with "..." which we use elsewhere when things like traces need to be truncated to 10K chars